lib/grover/js/processor.js

```diff
@@ -101,6 +101,18 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       }
     }
 
+    // add styles (if provided)
+    const addStyleTag = options.addStyleTag; delete options.addStyleTag;
```

Comment:
this should likely also be `styleTagOptions` (similarly for the script tag) given that's what they are..

Reply:
it'd possibly also be an array (which would be enumerated to allow multiple style/script tag options)

Reply:
I used the names from the puppeteer doc, but agree to changing it to reflect what they really are. I'll add entries to README too.

> it'd possibly also be an array (which would be enumerated to allow multiple style/script tag options)

True, it is indeed an array. I'll update it.